recipes/pyrdl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyrdl" %}
+{% set version = "1.1.3_rdkit" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 944b5816712a48bbf88aa25d4300ce11871ddf6e971218eac08f90ed2192f715
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python
+    - cython
+  host:
+    - python
+    - cython
+  run:
+    - python
+
+test:
+  source_files:
+    - 'build/*'
+
+  imports:
+    - py_rdl
```

Comment:
Are you using the `build/*` files in the import test? If so this package will be broken b/c the test files are not package. If not you can drop the test files b/c I don't see other tests that are using them.